docs/src/pages/components/slider/CustomizedSlider.js

```diff
@@ -178,39 +171,45 @@ function AirbnbThumbComponent(props) {
 
 export default function CustomizedSlider() {
   const classes = useStyles();
+  // For some reason the theme when styled used twice is coming from emotion, not our defulat theme
+  const theme = useTheme();
```

Comment:
Did we resolve this issue?

Reply:
Good catch, it was resolved. I've updated the examples to not contain these.